oak-store-composite/src/main/java/org/apache/jackrabbit/oak/composite/package-info.java

```diff
@@ -55,7 +55,7 @@
  *  This is obviously correct but may be slow.
  *  {@link org.apache.jackrabbit.oak.composite.CompositionContext#getContributingStores(java.lang.String, java.util.function.Function)}
  */
-@Version("0.3.0")
+@Version("0.4.0")
```

Comment:
According to guideline below, this should be 1.0.0
https://semver.org/#how-do-i-know-when-to-release-100

Reply:
As discussed, bumping now to 1.0.0 could break user code. Best done in a separate PR.